### PR TITLE
Remove spinner

### PR DESCRIPTION
**Pull request by benwaffle**

### CODE REVIEW

File: data/scripts/valadoc.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ var content_data = null;
 var RESULTS_BULK = 20;
 
 function close_spinner () {
```
Comment (danrabbit) [inline data/scripts/valadoc.js]: Shouldn't we remove this function if we're removing the spinner completely?

**Comment by benwaffle** [comment]
the indentation is screwed up

**Comment by btkostner** [comment]
I'll do a javascript lint pr after this

**Comment by benwaffle** [comment]
ok to merge?

**Pull request merged**